src/trust-region/trust-region.jl

```diff
@@ -24,18 +24,17 @@ and the following function:
 """
 @compat abstract type AbstractTrustRegion end
 
-"""`ρ = ratio(nlp, f, f_trial, Δm, x_trial, step, slope)`
+"""`ared, pred = aredpred(nlp, f, f_trial, Δm, x_trial, step, slope)`
```

Comment:
`ratio(nlp...)` is used by TRON, so tron is failing (line 41 of tron-trust-region.jl).